.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
I agree. As you may know unrar has license restrictions. I'm not aware if there's another option to avoid the unrar utility/library dependency.

https://rarfile.readthedocs.org/en/latest/faq.html